exporter/loggingexporter/internal/otlptext/logs.go

```diff
@@ -47,7 +47,8 @@ func (textLogsMarshaler) MarshalLogs(ld plog.Logs) ([]byte, error) {
 				lr := logs.At(k)
 				buf.logEntry("ObservedTimestamp: %s", lr.ObservedTimestamp())
 				buf.logEntry("Timestamp: %s", lr.Timestamp())
-				buf.logEntry("Severity: %s", lr.SeverityText())
+				buf.logEntry("SeverityTest: %s", lr.SeverityText())
```

Comment:
Text not test?

Reply:
Typo, Thanks for catching it :-D Fixed in https://github.com/open-telemetry/opentelemetry-collector/commit/05f0cc16da010ceff24ff61b0ba009fdc1d54314